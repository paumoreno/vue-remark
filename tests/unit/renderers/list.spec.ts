import VueRemarkList from '@/renderers/list.vue';
import { mount } from '@vue/test-utils';

describe(VueRemarkList.name, () => {
  test('snapshot (unordered)', () => {
    const wrapper = mount(VueRemarkList, {
      propsData: {
        ordered: false
      },
      slots: {
        default: `<li>some list item</li>`
      }
    });

    expect(wrapper).toMatchSnapshot();
  });

  test('snapshot (ordered)', () => {
    const wrapper = mount(VueRemarkList, {
      propsData: {
        ordered: true
      },
      slots: {
        default: `<li>some list item</li>`
      }
    });

    expect(wrapper).toMatchSnapshot();
  });
});
