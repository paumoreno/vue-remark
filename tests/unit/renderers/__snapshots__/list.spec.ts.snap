// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VueRemarkList snapshot (ordered) 1`] = `
<ol>
  <li>some list item</li>
</ol>
`;

exports[`VueRemarkList snapshot (unordered) 1`] = `
<ul>
  <li>some list item</li>
</ul>
`;
